modules/campaign_lib.py

```diff
@@ -358,7 +358,7 @@ def __call__(self,room,subnodes):
             debug.debug("Successfully added "+str(numsofar))
             numadded=0
             if (numsofar<self.cargnum):
-                rang=list[list(range(you.numCargo()))]
+                rang=list(range(you.numCargo()))
                 rang.reverse()
                 for i in rang:
```

Comment:
Okay, what you really need here is the following instead of the above 3 lines:

```python3
for i in range(you.numCargo(), 0, -1):
```

Basically:
- Line 361 is creating a list of numbers
- Line 362 is reversing it
- Line 363 is then traversing the list

We can actually merge these three things into one single, succinct statement in the above `for...range`.

Reply:
Nice catch @BenjamenMeyer